lib/cluster.js

```diff
@@ -728,6 +728,14 @@ function sendHelper(proc, message, handle, cb) {
 // to the callback but intercepts and redirects ACK messages.
 function internal(worker, cb) {
   return function(message, handle) {
+    if (message.cmd === 'NODE_WARN') {
```

Comment:
To make sure that warnings that occur within one of the cluster forks do not go unnoticed. The warnings emitted by the fork are forwarded to the parent process where they can be centrally handled.